docs/swagger.yaml

```
@@ -41,6 +41,35 @@ paths:
       summary: User login
       tags:
       - login
+  /refresh:
+    post:
+      consumes:
+      - application/x-www-form-urlencoded
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)